doc/multisig-tutorial.md

```diff
@@ -0,0 +1,225 @@
+# 1. Multisig Tutorial
+
+Currently, it is possible to create a multig wallet using Bitcoin Core only.
+
+Although there is already a brief explanation and a functional test about the multisig implemented in [PR 22067](https://github.com/bitcoin/bitcoin/pull/22067), this tutorial proposes to use the signet (instead of regtest), bringing the reader closer to a real environment and explaining some functions in more detail.
```

Comment:
Is `regtest` vs `signet` the main difference? I'd expect the guide to have the same steps regardless of the chain used. 

What's the long-term plan for this guide and the guide from #22067? Do we plan to maintain both of them or keep only one?

Reply:
The "Basic multisig example" in 22067 should probably link to this guide, but let's see what gets merged first.